Framework/Beamline/src/DetectorInfo.cpp

```diff
@@ -102,6 +102,12 @@ size_t DetectorInfo::scanSize() const {
   return m_positions->size();
 }
 
+/**
+ * Returns true if all of the detectors all have the same scan interval. Will
+ * return false if DetectorInfo is not scanning.
```

Comment:
I am now thinking that `true` would make more sense since no scanning implies that all intervals are identical, i.e., sync?

Reply:
I have been back and forth with this. I think the name implies the current behaviour, i.e. it is a scan with identical time indexes. We could have `hasIdenticalScanIntervals` instead?

The question would be what the more common use would be? In `NormaliseToMonitor` I would then `isScanning && hasIdenticalScanIntervals` or similar. Perhaps this is clearer?

Reply:
Maybe what we should do is avoid overlap with the `isScanning()` method and make the naming convention agree?

Maybe `isScanningSync()`? Well actually that is also unclear, depending on whether "sync" stands for "synchronous" or "synchronously" it could be implying `isScanning` or not? So then in long form `isScanningSynchronously()` to make it clear?

Your suggestions also sounds ok (even though "identical" leaves a it a bit unclear what is identical with what, it could also be taken to mean that all intervals have the same length or something). To decide which option you go with maybe ask yourself whether most client code would be interested in identical scan intervals, or in actual scanning.

Reply:
As per Slack conversation we will keep this as `isSyncScan` with the current behaviour. There is no perfect option, but this does offer all the functionality needed.